### PR TITLE
Add sickle cell

### PR DESCRIPTION
**Pull request by bonjourmauko**

### CODE REVIEW

File: covid19-algorithme-orientation-openapi.json [hunk 1]
```diff
@@ -245,14 +238,13 @@
             },
             "risk_factors": {
               "breathing_disease": true,
-              "cancer": true,
```
Comment (bzg) [inline covid19-algorithme-orientation-openapi.json]: Why the "cancer" line has been removed here?

File: lib/covid19_questionnaire_web/schemas/questionnaire_request.ex [hunk 2]
```diff
@@ -45,15 +45,13 @@ defmodule Covid19QuestionnaireWeb.Schemas.QuestionnaireRequest do
           "liver_disease" => true,
           "diabetes" => true,
           "immunosuppressant_disease" => 1,
-          "immunosuppressant_drug" => 1,
-          "cancer" => true,
```
Comment (bzg) [inline lib/covid19_questionnaire_web/schemas/questionnaire_request.ex:49]: Same comment for "cancer": the frontend still send this variable.

**Review by bzg — CHANGES_REQUESTED** [review]
There is only the issue about "cancer" lines.

File: covid19-algorithme-orientation-openapi.json (hunk 1 above)
Comment (bonjourmauko) [reply covid19-algorithme-orientation-openapi.json]: oups !

File: lib/covid19_questionnaire_web/schemas/questionnaire_request.ex (hunk 2 above)
Comment (bonjourmauko) [reply lib/covid19_questionnaire_web/schemas/questionnaire_request.ex:49]: Corrected, thanks !

**Review by bzg — APPROVED** [review]
Great, thanks for fixing this, it's good for me.

**Comment by bzg** [comment]
(I let you merge this!)

**Comment by bonjourmauko** [comment]
Done ! @bzg call me ☎ for deploymenty

**Comment by bzg** [comment]
Thanks! I will :phone: you later on.

**Pull request merged**